cg-spec/editors_draft.html

```diff
@@ -375,6 +375,21 @@ <h2>Overview</h2>
 -->
 </pre>
 
+    </section>
+    <section>
+      <h2>Common misconceptions</h2>
+
+      <p>People coming to RDF-star may have wrong ideas about what it is or how to use it. The purpose of this section is to dispel the most common misconceptions.
+
+      <p>RDF-star is <em>not</em> syntactic sugar for <a href="https://www.w3.org/TR/rdf11-mt/#reification">standard reification</a>. While the latter is a vocabulary fitting in the standard RDF model (abstract syntax), RDF-star extends that model (see <a href="#concepts"></a>) with a new construct, namely <a>quoted triples</a>. Note also that the two are not mutually exclusive: standard reification can still be used in conjunction with RDF-star.
+
+      <p>Unlike reified statements, RDF-star <a>quoted triples</a> are unique: wherever `&lt;&lt; :employee38 :jobTitle "Assistant Designer" &gt;&gt;` appears, it denotes one and the same thing. This impacts the modelling choices one has to make when using RDF-star; this is further discussed in <a href="#occurrences"></a>.
```

Comment:
```suggestion
      <p>Unlike reified statements, but like URIs, RDF-star <a>quoted triples</a> are unique: wherever `&lt;&lt; :employee38 :jobTitle "Assistant Designer" &gt;&gt;` appears, it always denotes one and the same thing. This impacts the modeling choices one has to make when using RDF-star; this is further discussed in <a href="#occurrences"></a>.
```

Reply:
```suggestion
      <p>Unlike reified statements, RDF-star <a>quoted triples</a> are unique: wherever `&lt;&lt; :employee38 :jobTitle "Assistant Designer" &gt;&gt;` appears, it always denotes one and the same thing. This impacts the modeling choices one has to make when using RDF-star; this is further discussed in <a href="#occurrences"></a>.
```

Reply:
@TallTed I didn't keep the "but like URIs" from your suggestion above. First, because it should be "IRIs" :wink:. Second, because the reference to IRIs rather than, say, literals, might be confusing. Finally, because the comparison of quoted triples with IRIs (or literals, for that matter) becomes slippery when quoted triples contain blank nodes (although of course, that does not change anything to their "unicity" described here).